### PR TITLE
Create how-to-restart-and-debug.md

### PR DESCRIPTION
**Pull request by Yehonal**
## Description
[This post on StackOverflow](https://stackoverflow.com/questions/62910364/azerothcore-with-gdb-restarter-and-crashlog) is complete enough to be moved on our wiki as a proper guide on how to restart and debug with gdb

### CODE REVIEW

File: docs/how-to-restart-and-debug.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+It should be enough to automatically restart, monitoring and gdb your server
+
+### USING DOCKER
+
+Work in progress...
+
+
+  [1]: https://i.stack.imgur.com/EyIi7.png
+  [2]: https://pm2.keymetrics.io/
```
Comment (FrancescoBorzi) [inline docs/how-to-restart-and-debug.md:59]: shall we add this page somewhere?

Comment (Yehonal) [reply docs/how-to-restart-and-debug.md:59]: Yes, any suggestions?

Comment (FrancescoBorzi) [reply docs/how-to-restart-and-debug.md:59]: no suggestions :)

Comment (Yehonal) [reply docs/how-to-restart-and-debug.md:76]: Linked into the documentation index page

**Pull request merged**